src/state/reducer/rootReducer.js

```diff
@@ -0,0 +1,20 @@
+import initialState from '../store/initialState'
+
+const rootReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "STORY":
+      return {
+        ...state,
+        articleType: 'story'
+      }
+    case "EXPERIENCE":
+      debugger
+      return {
+        ...state,
+        articleType: 'experience'
```

Comment:
refactor this to one type where you dynamically set the article type depending on the payload